app.go

```diff
@@ -302,6 +302,11 @@ type Static struct {
 	//
 	// Optional. Default value 0.
 	MaxAge int `json:"max_age"`
+
+	// Next defines a function to skip this middleware when returned true.
+	//
+	// Optional. Default: nil
+	Next func(c *Ctx) bool
```

Comment:
Could you change this to the `Handler` type?

Reply:
Assuming that you're talking about this `Handler` type:
```go
// Handler defines a function to serve HTTP requests.
type Handler = func(*Ctx) error
```
I'm not sure under what circumstances the middleware should be skipped.
If `Handler` returns `nil` or `error`? 

Here is current login of `app.Static`:
```go
// Don't execute middleware if Next returns true
if config != nil && config[0].Next != nil && config[0].Next(c) {
    return c.Next()
}
// Serve file
// ...
```